sys-libs/readline/readline-6.3.8.recipe

```diff
@@ -85,6 +89,7 @@ BUILD()
 INSTALL()
 {
 	make install
+	strip -s $libDir/libhistory.so.6.3 $libDir/libreadline.so.6.3
```

Comment:
Do you mean replacing "strip" in the highlighted green line (line 92) with "strip$secondaryArchSuffix"? Or do you mean replacing the entire line? I tried the latter, and it didn't work. (echoing "strip$secondaryArchSuffix" shows "strip" because $secondaryArchSuffix is null.)